Sources/LiveViewNative/Modifiers/ForegroundStyleModifier.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  ForegroundStyleModifier.swift
+// LiveViewNative
+//
+//  Created by May Matyi on 2/17/23.
+//
+
+import SwiftUI
+
+struct ForegroundStyleModifier: ViewModifier, Decodable {
+    private let primary: WrappedShapeStyle?
+
+    init(from decoder: Decoder) throws {
+        let container = try decoder.container(keyedBy: CodingKeys.self)
+
+        self.primary = try container.decodeIfPresent(WrappedShapeStyle.self, forKey: .primary)
```

Comment:
You can make `primary` non-optional to avoid the force unwrap in `body(content:)`
```suggestion
        self.primary = try container.decode(WrappedShapeStyle.self, forKey: .primary)
```

Reply:
Good call. Fixed